submissions/currency_converter/apifetch.js

```diff
@@ -0,0 +1,21 @@
+// import  APIKEY from "./key";
+
+const convert = () => { 
+    const countrySymbol = document.getElementById('symbol').value; 
+    const amountInUSD = document.getElementById('amount').value;
+
+const options = { 
+    method: 'GET',  
+    headers: { 
+        accept: 'application-json',
+        // 'X-API-KEY': APIKEY
+        'X-API-KEY' : "f53b14db7658d93aeb2f8589838b87e01d730d82b9c73e9878065452902790b7"
+    }
+};
+    const apiUrl = 'https://api.chimoney.io/v0.2/info/usd-amount-in-local?destinationCurrency=USD&amountInUSD=1' 
```

Comment:
The issue is that you’re using the prod endpoint instead of staging. You can switch between prod and staging endpoint in the api docs

also consider managing api keys, chimoney base url and other changeable parameters in env variable

Reply:
cant find any resources about the prod and staging endpoints can we have a live meeting you show me the way around?